gui/src/2D/controls/container.ts

```diff
@@ -23,6 +26,10 @@ export class Container extends Control {
     protected _adaptWidthToChildren = false;
     /** @hidden */
     protected _adaptHeightToChildren = false;
+    /** @hidden */
+    protected _intermediateTexture: Nullable<DynamicTexture> = null;
+    /** @hidden */
+    public _renderToIntermediateTexture: boolean = false;

```

Comment:
Can't you remove the leading underscore (and the `hidden` modifier) and let the property be updatable by the user at any point? I think with the current code it should work?

Reply:
Needed to make a slight change, as if we changed this property to false as it was we would still render to the intermediate texture if it had been created